### PR TITLE
Docs: fixed why-keystone heading and first content block on small screen

### PR DESCRIPTION
**Pull request by mikehazell**
On the "Why keystone" page the heading was being clipped and the first content block was in 2 very narrow columns on smaller screens.

| Before | After |
|:----|:----|
| image |image|

### CODE REVIEW

File: docs/pages/why-keystone.tsx [hunk 1]
```diff
@@ -40,17 +40,17 @@ export default function WhyKeystonePage() {
   return (
     <Page>
       <MWrapper>
-        <Type as="h1" look="heading92">
+        <Type as="h1" look="heading92" fontSize={['4rem', '5.75rem']}>
```
Comment (mikehazell) [inline docs/pages/why-keystone.tsx:43]: I copied these sizes from `docs/components/primitives/Type.tsx`

Not sure if there's a more Design-Token-y way of writing this.

Comment (JedWatson) [reply docs/pages/why-keystone.tsx:43]: Arguably we should possibly do this for all the headings with this `look`?

It's defined here: https://github.com/keystonejs/keystone/blob/master/docs/components/primitives/Type.tsx#L68-L74

Or do you think this fix should be local to this page?

**Review by raveling — COMMENTED** [review]
looks good!

File: docs/pages/why-keystone.tsx (hunk 1 above)
Comment (raveling) [reply docs/pages/why-keystone.tsx]: I opted to use the `<IntroHeading>` component Dominik setup on other pages which has all the responsive behaviour baked in.
hope that's cool

**Review by raveling — APPROVED** [review]

**Pull request merged**